pkg/endpoint/builder.go

```diff
@@ -33,6 +33,7 @@ type endpointBuild struct {
 }
 
 func (e *Endpoint) newEndpointBuild(owner Owner, context *RegenerationContext) *endpointBuild {
+	context.Stats.queueWait.Start()
```

Comment:
If anyone ever passes a nil `RegenerationContext`, this may panic.

Reply:
We already rely on context not being nil across the entire regeneration code. What do you suggest to do if it is nil? Do you want to add it to all functions?

Reply:
Given that we control the value of `context`, this shouldn't be an issue. I was speaking from prior experience with dereferencing of pointers directly, d2c0887afc8843e9c38df1da24ef53eca67dbd53 .